### ISSUE TITLE
color contrast

### ISSUE THREAD

**Issue by kisram** [emt-project/emt-static#155]
try to find color combinations that have enough contrast to comply with WCAG

**Comment by kisram** [comment]
For the entities, I'm thinking of getting rid of the inverted colors (entity color background + white text) because it just doesn't work well with the cases where the text ends up being blue instead of white because there's an abbreviation or sth else as well. So a lighter entity color highlight + normal (or blue) text. This would mean institutions probably need an entirely new color though.

**Comment by kisram** [comment]
The only remaining contrast warning I've come across is for the invisible transcript line numbers and I'll figure out what to do about the line numbers later as part of the accessibility issue
